CefGlue.Common/build/CefGlue.Common.targets

```
@@ -11,6 +11,36 @@
             <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
             <PublishState>Included</PublishState>
         </None>
+    </ItemGroup>
+        <ItemGroup Condition="'$(CefGlueTargetPlatform)' == 'linux'">
+        <None Include="@(CefRedistOSX64)">
```

```suggestion
        <None Include="@(CefRedistLinux64)">
```

Also needs to be changed here:
https://github.com/pheonixfirewingz/cef.redist.linux/blob/main/build/cef.redist.linux64.props#L4